### PR TITLE
Updating screenshots for kiali task

### PR DESCRIPTION
**Pull request by xeviknal**
Updating observability task with screenshots from kiali version used in Istio 1.6

### CODE REVIEW

**Comment by xeviknal** [comment]
@jmazzitelli do you mind checking this out?

File: content/en/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ If you haven't already applied destination rules, follow the instructions in [Ap
 1.  To get started, run this command to route all traffic to the `v1` version of
 each microservice.
 
-    {{< text syntax=bash snip_id=config_all_v1 >}}
+    {{< text syntax=bash snip_id="config_all_v1" >}}
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/traffic-shifting/index.md:39]: Why is these snip_ids being changed?

File: content/en/docs/tasks/traffic-management/traffic-shifting/index.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ article [Canary Deployments using Istio](/blog/2017/0.1-canary/).
 
 1. Remove the application routing rules:
 
-    {{< text syntax=bash snip_id=cleanup >}}
+    {{< text syntax=bash snip_id="cleanup">}}
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/traffic-shifting/index.md:116]: Missing " " (space) character. UI and snippet generator both require code blocks to have space before `>}}`.

**Comment by jmazzitelli** [comment]
The images look good. You probably have to run this through some linter/generator to get the tests to pass.

File: content/en/docs/tasks/traffic-management/traffic-shifting/index.md (hunk 2 above)
Comment (xeviknal) [reply content/en/docs/tasks/traffic-management/traffic-shifting/index.md:116]: oh yeah. Forgot to remove this. In my local, hugo complained about that and had to change it. Undoing this addition.

**Comment by xeviknal** [comment]
@frankbu do you mind assisting me for passing the tests? I don't know if I am in front of a glitch on the CI or if I am missing some proc to do. thanks

**Pull request merged**